### ISSUE TITLE
Ignore Sending Notifications for Own Actions

### ISSUE THREAD

**Issue by tgmcguire** [integrations/slack#472]
Hi there,

I am new to the GitHub + Slack integration, and was thinking that it would be great if Slack didn't notify me about my own activity.

For example, I added a few private repos into my "conversation" with the GitHub app. I merged a pull request, and received a notification for it – but since this was my own action, I really didn't need to be notified about it.

Any way around this, or is there any possibility this functionality/option could be worked in?

Thanks!!

**Comment by bkeepers** [comment]
Hey @tgmcguire, thanks for the feature request! 

This is definitely something that would be possible. We don't currently do anything special to handle notifications in DMs vs other channels, but I think it makes sense to not deliver your own activity to you in a DM.

That being said, this isn't a priority for us, but we'll leave this issue open in case anyone else wants to try implementing it.

**Comment by DiesIrae** [comment]
It's still the main pain-point IMHO, would love to see this fixed!

**Comment by carlbennettnz** [comment]
Yeah, this is really annoying. Would be great to see this reopened.

**Comment by rdxroman** [comment]
Would be nice to not get notifications for my own commits!

**Comment by Visput** [comment]
Could this be reopen?
Really annoying to be notified about my own actions. 
It could work similarly how GitHub email notifications work with setting "Include your own updates" being off.

**Comment by spydercapriani** [comment]
Agreed, it would be nice to filter out some of the spam I create with commits 👍

**Comment by martlume-operose** [comment]
+1

**Comment by markypython** [comment]
+1

**Comment by scepticulous** [comment]
Hey 👋 ,
thanks for bringing this to our attention.
A quick thing that would help us triage:
@markypython @martlume-operose @Visput (and everybody else) could you please vote on the original post with 👍 instead of adding a new comment. We are using this as an indicator of the existing demand.

**Comment by spydercapriani** [comment]
👍

**Comment by BruceWangTRT** [comment]
👍

**Comment by Niaro** [comment]
Wow can't believe its been 2 years and there is no way to filter out your own actions. When I make a PR review with 20+ comments, I get notified 20+ times... that's extremely annoying.

**Comment by thesyntaxinator** [comment]
Bump

**Comment by mr3m** [comment]
for example `github unsubscribe owner/repo comments:user_name`

**Comment by shizukann** [comment]
+1 from 2021, indeed annoying to receive notifications from own actions

**Comment by rbroggi** [comment]
+1

**Comment by eytanhanig** [comment]
> "All that is left for us at this point is for one of us to do the work and submit a PR for this."

@PeterHewat This repo no longer accepts PRs. From [the Readme](https://github.com/integrations/slack/blob/master/README.md#github--slack-integration) (emphasis mine):

> "**This repository is not accepting any code contributions.** The current code which is running for GitHub and Slack integration has significantly diverged from the code present in this repository as it contains specific code which is required to run service in GitHub infrastructure and which can not be open sourced at this point of time."

**Comment by sohnya** [comment]
I reacted to the original post, but I'm wondering if anyone is actually getting/reading those notifications?

**Comment by sewi-cpan** [comment]
This is very annoying. Will this issue ever be solved?

**Comment by mxcl** [comment]
“I know, I’ll install the GitHub slack app so I can get notifications for my team’s activity on GitHub”

*is overwhelmed by his own activity*

> mxcl uninstalls the app.

GitHub needs to be replaced, a startup would not make this crap.

**Comment by ahaverty** [comment]
Just to add why this is a impactful bug for me:
1. I'm focused on a PR
2. I leave a comment or review for that PR
3. Alarm bells and visual distractions immediately spring onto screen 🔔 "Who is pinging me on slack!?, why is the slack app bouncing on my screen". Better check in case services have gone down, of it it's important.
4. Open Github bot, and notice it's Slack telling me You have left a comment on a PR!.. 🙃 
5. Lose track of what I was doing..
6. Eventually regain context after 15 minutes of wondering how this is still an issue, and whether today will be the day I comment "+1" on this issue.

**Comment by fogx** [comment]
How many upvotes does a feature request require to be "a priority"? Over 100 should be more than enough :thinking:

**Comment by em843** [comment]
Still awaiting this feature eagerly. Axolo seems cool, but is too involved a solution for my small team.

**Comment by Noyabronok** [comment]
This is the airport, and I would like to announce my departure from GitHub Slack integration.  Way too difficult/impossible to configure it just for what I need.  At a basic level, I just want to get notified when someone mentions me at an org level.  Instead, I have to subscribe per repo to get spammed with my own actions.  Byeeee!

**Comment by CarlosLopezRoa** [comment]
At this point this is a 6 year+ running joke 🙃

**Comment by DevinWalker** [comment]
Chiming in with my request for this feature. Please, Lord, build it.

**Comment by peter-8640** [comment]
+1 🥱

**Comment by michael-hewitt** [comment]
Missing pull requests from other developers due to noise from my own pull requests.  Please add the ability to ignore one's own PRs in the Slack integration.

**Comment by Ptijohn** [comment]
Not sure if that would help anyone, but have you tried the Scheduled reminders > Real-time alerts functionality on the Github side? 

![image](https://github.com/user-attachments/assets/33e16d7b-b347-45c3-8083-3b8bcb9e577d)

This has basically all the similar functionalities regarding receiving Github notifications related to your activity, but without receiving messages about your own comments ;)
